core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -18,19 +18,33 @@
 import alluxio.exception.ExceptionMessage;
 
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 /**
  * Tests enum type {@link PropertyKey}.
  */
 public final class PropertyKeyTest {
+  private PropertyKey mTestProperty;
+  private PropertyKey mNullAliasTestProperty;
 
+  @Before
+  public void initialize() {
+    mTestProperty = PropertyKey.create("alluxio.test.property", false,
```

Comment:
NIT: for `mTestProperty` and `mNullAliasTestProperty`, you can initialize them directly along with the definition without adding this `Before` method. e.g.:

```java
  private PropertyKey mTestProperty = PropertyKey.create("alluxio.test.property", false,
      new String[] {"alluxio.test.property.alias1", "alluxio.test.property.alias2"});
  private PropertyKey mNullAliasTestProperty = 
      PropertyKey.create("alluxio.test.property.nullAlias", false, null);
```